### PR TITLE
chore: keeping spelling consistencies same. `GitHub` and `YouTube` trademarks

### PR DESCRIPTION
**Pull request by davorpa**
## What does this PR do?
Improve repo

### Why is this valuable (or not)?

Solves trademarks typos using the text provided by the company

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [x] Search for duplicates. #5940
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
If a title at the source spells it "Github" we shouldn't be changing it to "GitHub'

**Comment by davorpa** [comment]
> If a title at the source spells it "Github" we shouldn't be changing it to "GitHub'

Thanks for comment in a draft. It was replaced by regex. A deep review for all those...

:ok: * [How to Collaborate on Github](https://github.com/eonist/How-to-collaborate-on-github) - André J
:no_entry: * [Introduction to Git and GitHub](https://launchschool.com/books/git) - Launch School
:ok: * [Introduction to Git and GitHub - Tutorial](http://cse.unl.edu/~cbourke/gitTutorial.pdf) - Dr. Chris Bourke (PDF)
:ok: * [Unix System Programming in OCaml](http://ocaml.github.io/ocamlunix/) [GitHub Repo](https://github.com/ocaml/ocamlunix/)
:no_entry: * [GitHub帮助文档](https://github.com/waylau/github-help)
:no_entry: * [Jhan Carlos Silva - Git and Github Tutorial](https://www.youtube.com/watch?v=NPRUsCcZwJ4) (screencast)
:no_entry: * [Git and GitHub&rlm;](https://www.youtube.com/playlist?list=PLDoPjvoNmBAw4eOj58MZPakHjaO3frVMF) - Elzero Web School
:no_entry: * [Git and GitHub in One Video (Theory + Practical) \| A 2 Z in Bangla](https://www.youtube.com/watch?v=4KdGgGsIDeA&t=787s) - SHAJ.T3CH
:arrows_clockwise: * [Machine Learning with Python - YouTube Playlist](https://www.youtube.com/playlist?list=PLZoTAELRMXVPBTrWtJkn3wWQxZkmTXGwe) - Krish Naik (YouTube)
:arrows_clockwise: * [Git & GitHub](https://www.youtube.com/playlist?list=PLFIM0718LjIVknj6sgsSceMqlq242-jNf) - Sandhika Galih, Web Programming UNPAS (YouTube)
:no_entry: * [Cách sử dụng Git trong THỰC TẾ \| GitHub](https://www.youtube.com/watch?v=O5uT6p6VWjY) - Ông Dev
:no_entry: * [Từ gà tới pro Git và GitHub trong 20 phút - Tự học Git siêu tốc](https://www.youtube.com/watch?v=1JuYQgpbrW0) - Phạm Huy Hoàng (toidicodedao)

where: 
- :ok: => needing change
- :arrows_clockwise: => title change / rewording
- :no_entry: => revert

**Review by CleanMachine1 — APPROVED** [review]

**Review by LuigiImVector — APPROVED** [review]

File: casts/free-podcasts-screencasts-en.md [hunk 1]
```diff
@@ -110,7 +110,7 @@
 
 * [All Things Git](https://www.allthingsgit.com) - Edward Thomson and Martin Woodward (podcast)
 * [GitMinutes](http://www.gitminutes.com) (podcast)
-* [Jhan Carlos Silva - Git and Github Tutorial](https://www.youtube.com/watch?v=NPRUsCcZwJ4) (screencast)
+* [Jhan Carlos Silva - Git and GitHub Tutorial](https://www.youtube.com/watch?v=NPRUsCcZwJ4) (screencast)
```
Comment (eshellman) [inline casts/free-podcasts-screencasts-en.md:113]: source uses 'Github'

File: courses/free-courses-pt_BR.md [hunk 2]
```diff
@@ -97,14 +97,14 @@
...
-* [Curso COMPLETO de Flutter](https://youtube.com/playlist?list=PLlBnICoI-g-d-J57QIz6Tx5xtUDGQdBFB) - Flutterando (Youtube)
+* [Curso COMPLETO de Flutter](https://youtube.com/playlist?list=PLlBnICoI-g-d-J57QIz6Tx5xtUDGQdBFB) - Flutterando (YouTube)
 
 
 ### Git
 
 * [Curso de Git e GitHub: grátis, prático e sem usar comandos no terminal](https://www.youtube.com/playlist?list=PLHz_AreHm4dm7ZULPAmadvNhH6vk9oNZA) - Gustavo Guanabara (YouTube)
 * [Git e contribuições para projetos Open Source](https://www.udemy.com/course/git-e-github/) - Bruno Orlandi (Udemy)
-* [Git e Github para iniciantes](https://www.udemy.com/git-e-github-para-iniciantes/) - Willian Justen de Vasconcellos (Udemy)
+* [Git e GitHub para iniciantes](https://www.udemy.com/git-e-github-para-iniciantes/) - Willian Justen de Vasconcellos (Udemy)
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:107]: source uses 'Github'

**Review by eshellman — CHANGES_REQUESTED** [review]

File: casts/free-podcasts-screencasts-en.md (hunk 1 above)
Comment (davorpa) [reply casts/free-podcasts-screencasts-en.md]: Reverted in 9d12e9596a95dc45ea0370a8258e5fad03c8477a

File: courses/free-courses-pt_BR.md (hunk 2 above)
Comment (davorpa) [reply courses/free-courses-pt_BR.md]: Reverted in 9d12e9596a95dc45ea0370a8258e5fad03c8477a

**Pull request merged**